db/schema.rb

```diff
@@ -929,6 +927,8 @@
     t.boolean "no_emails", default: false, null: false
     t.string "inat_username"
     t.integer "original_image_quota", default: 0
+    t.integer "hide_authors"
```

Comment:
Is losing the `null: false` here intentional?  I'm guessing this causes the default to be `nil` rather than `0`.